java/daikon/dcomp/DCInstrument.java

```diff
@@ -722,6 +737,24 @@ public JavaClass instrument() {
     return gen.getJavaClass().copy();
   }
 
+  /**
+   * Returns true if the specified classname.method_name is the root of JUnit startup code.
+   *
+   * @param classname class to be checked
+   * @param method_name method to be checked
+   * @return true if the given method is a JUnit trigger
+   */
+  boolean is_junit_trigger(String classname, String method_name) {
```

Comment:
Minor:  stylistic Java code uses camelCaseVariables.  Python and C use snake_case_variables.  I don't really care, but code is a touch easier to read if it all uses a similar convention.